ruby/ruby/IceLocal/LocalExceptions.rb

```diff
@@ -0,0 +1,159 @@
+# encoding: utf-8
+#
+# Copyright (c) ZeroC, Inc.
+
+# This file provides all the exceptions derived from Ice::LocalException.
```

Comment:
All these exceptions are constructed from the C++ code. Not one is raised by the Ruby code.